staging/volumes/azure_file/README.md

```diff
@@ -6,15 +6,32 @@ Install *cifs-utils* on the Kubernetes host. For example, on Fedora based Linux
 
 Note, as explained in [Azure File Storage for Linux](https://azure.microsoft.com/en-us/documentation/articles/storage-how-to-use-files-linux/), the Linux hosts and the file share must be in the same Azure region.
 
-Obtain an Microsoft Azure storage account and create a [secret](secret/azure-secret.yaml) that contains the base64 encoded Azure Storage account name and key. In the secret file, base64-encode Azure Storage account name and pair it with name *azurestorageaccountname*, and base64-encode Azure Storage access key and pair it with name *azurestorageaccountkey*.
+## Create a storage access secret
+
+Obtain an Microsoft Azure storage account and create a [secret](secret/azure-secret.yaml) that contains the base64 encoded Azure Storage account name and key. In the secret file, base64-encode Azure Storage account name and pair it with name `azurestorageaccountname`, and base64-encode Azure Storage access key and pair it with name `azurestorageaccountkey`.
+
+Alternatively, use `kubectl` directly to create the secret:
```

Comment:
I would suggest use `kubectl create secret` command as first choice

Reply:
I agree. I just didn't want to change what was there before, just add things. But I will rewrite it slightly.

Reply:
maybe you could change the suggestion sequence :)

Reply:
Did so ;-)